e2e/app/element/base-element.ts

```diff
@@ -151,10 +151,33 @@ export default class BaseElement extends Container {
       });
   }
 
-  async type(text: string, options?: { delay: number }): Promise<this> {
-    await this.focus();
-    await this.clear();
-    await this.asElementHandle().then((handle: ElementHandle) => handle.type(text, options));
+  /**
+   * Clear existing value in textbox then type new text value.
+   * @param {string} The texts string.
+   * @param options The typing options.
+   */
+  async type(textValue: string, options?: { delay: number }): Promise<this> {
```

Comment:
While running `dataset-create.spec` to make sure it runs okay, I observed `clear()` func acts flaky. Old value in text input was not always cleared before type new text. 
My fix is to add one retry.